cluster-scope/base/core/namespaces/thoth-graph-prod/resourcequota.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
 spec:
   hard:
     limits.cpu: '12'
```

Comment:
when you increase `requests.cpu`, `limits.cpu` needs to increase as well. Requested resources cannot be more than the resource limits.

Reply:
very good point, it was mistake, fixing it :sweat_smile: